rllib/utils/test_utils.py

```diff
@@ -1622,7 +1622,17 @@ def run_rllib_example_script_experiment(
                 os.environ.get("TEST_OUTPUT_JSON", "/tmp/learning_test.json"),
                 "wt",
             ) as f:
-                json.dump(json_summary, f)
+                try:
+                    json.dump(json_summary, f)
+                # Something went wrong writing json. Try again w/ simplified stats.
+                except Exception:
```

Comment:
Do we need such an exception also directly in our JSON logger?

Reply:
Might already be handled there, I think. DreamerV3 runs fine the all the default loggers active (JSON, CSV, TBX), so I think we are good. This here is just to write the tiny metadata to a file (that gets then sent to S3 via our buildkite release pipeline) at the end of a release test.